### PR TITLE
More verbose modes for report --how display

### PR DESCRIPTION
**Pull request by lukaszachy**
-v    ... result + name
-vv   ... result + name + content of out.log
-vvv  ... result + name + content of all log files

### CODE REVIEW

**Comment by psss** [comment]
Nice idea! Perhaps one adjustment: What about showing log paths in verbose level 2 and the whole log content in level 3?

File: tmt/steps/report/display.py [hunk 1]
```diff
@@ -19,3 +21,13 @@ def go(self):
             return
         for result in self.step.plan.execute.results():
             self.verbose(result.show(), shift=1)
+            # -vv at least for files
+            if self.opt('verbose') > 1:
```
Comment (psss) [inline tmt/steps/report/display.py:25]: What about moving this whole section under a dedicated method?

File: tmt/steps/report/display.py [hunk 2]
```diff
@@ -19,3 +21,13 @@ def go(self):
...
+                for log_file in result.log:
+                    log_name = os.path.basename(log_file)
+                    # -vv for out.log, -vvv for all log files
+                    if self.opt('verbose') == 2 and log_name not in ['out.log']:
+                        continue
+                    real_log_path = os.path.abspath(self.workdir + "/../../execute/" + log_file)
+                    with open(real_log_path) as f:
+                        self.verbose(log_name,f.read(), shift=2)
```
Comment (psss) [inline tmt/steps/report/display.py:33]: Let's use `yellow` for the log name, to make is more easily foundable.

**Comment by lukaszachy** [comment]
Now it prints:
-v    ... result + name
-vv   ... result + name + path to logs
-vvv  ... result + name + path to logs + content of out.log

**Comment by psss** [comment]
Now looks great! Thanks much for the improvements. Added a couple of minor changes. Seems we'll need to improve `Common._indent()` to nicely handle multiline keys as well, for now added a `content` key, so that the full output is correctly indented.

**Pull request merged**